### PR TITLE
Allow evolution of fragmentation functions grids

### PR DESCRIPTION
**Pull request by t7phy**

### CODE REVIEW

**Comment by felixhekhorn** [comment]
Please make the PR title more useful

**Comment by t7phy** [comment]
> Please make the PR title more useful

Done

**Comment by felixhekhorn** [comment]
In case this is ready please ask for review

**Review by felixhekhorn — APPROVED** [review]

File: src/pineko/evolve.py [hunk 1]
```diff
@@ -157,6 +157,10 @@ def write_operator_card(pineappl_grid, default_card, card_path, tcard):
     if "polarized" in kv:
         operators_card["configs"]["polarized"] = kv["polarized"] == "True"
 
+    # fragmentation function grid?
+    if "timelike_evol" in kv:
+        operators_card["configs"]["timelike"] = kv["timelike_evol"] == "True"
```
Comment (felixhekhorn) [inline src/pineko/evolve.py:162]:
```suggestion
    if "timelike" in kv:
        operators_card["configs"]["timelike"] = kv["timelike"] == "True"
```

**Review by felixhekhorn — CHANGES_REQUESTED** [review]

**Comment by felixhekhorn** [comment]
> > Please make the PR title more useful
> 
> Done

There is no point in saying you're changing something to pineko in the pineko repo

**Pull request merged**